modules/retail/pages/retail-components.adoc

```diff
@@ -104,15 +107,33 @@ You can use formulas to apply configuration patterns to your hardware groups.
 
 You can use default settings for formulas, or edit them to make them more specific to your environment.
 
-For more information about formulas, see xref:retail:retail-formulas.adoc[].
+For more information about formulas, see xref:retail:retail-formulas-intro.adoc[].
 
 
 
-[[retail.sect.components.fitting.system-groups]]
-=== System groups
+=== Saltboot
 
-{smr} uses system groups to organize the various devices in your environment.
+Saltboot is a collection of tools and processes that are used to bootstrap, deploy and validate {smr} terminals.
 
-Each branch requires a system group, containing a single branch server, and the PoS terminals associated with that server.
-Each system group is identified with a Branch ID.
-The Branch ID is used in formulas and scripts to automatically update the entire group.
+Saltboot consists of:
+
+* Initialization:
++
+The saltboot ``initrd`` is created during image building and is required for bootstrapping terminals.
+
+* Saltboot state:
++
+The Salt state that contains the logic for the entire saltboot process.
+
+* Partitioning pillars:
++
+The Salt pillar structure that describes how terminals are partitioned and what image is deployed on each terminal.
+
+* Images and boot images pillars:
++
+These Salt pillars are generated after saltboot has successfully built an image.
```

Comment:
One last correction. Saltboot is not building the image. Image is build using SUMA image building. The difference between other images (e.g. for virtualization) and retail is that the retail images contain saltboot initrd.
SUMA image building detects that and when the image does have saltboot initrd it generates these salt pillars.

So the sentence should be hinting that these salt pillars are generated after image with saltboot initrd has been successfuly built.

Reply:
OK, I've tried again! Am I getting closer?